### ISSUE TITLE
Cypress failing to run suddenly

### ISSUE THREAD

**Issue by craig-dae** [cypress-io/github-action#144]
No changes were made to out pipeline. This has started happening today. May be related to an outage with GitHub actions that was occurring this morning. 

2020-05-20T17:31:59.0347538Z ##[warning]Unexpected input 'working-directory', valid inputs are ['record', 'config', 'config-file', 'env', 'browser', 'command']
2020-05-20T17:31:59.0351634Z ##[warning]Unexpected input 'group', valid inputs are ['record', 'config', 'config-file', 'env', 'browser', 'command']
2020-05-20T17:31:59.0352439Z ##[warning]Unexpected input 'headless', valid inputs are ['record', 'config', 'config-file', 'env', 'browser', 'command']
2020-05-20T17:31:59.0352958Z ##[warning]Unexpected input 'parallel', valid inputs are ['record', 'config', 'config-file', 'env', 'browser', 'command']
2020-05-20T17:31:59.0353472Z ##[warning]Unexpected input 'start', valid inputs are ['record', 'config', 'config-file', 'env', 'browser', 'command']
2020-05-20T17:31:59.0353983Z ##[warning]Unexpected input 'wait-on', valid inputs are ['record', 'config', 'config-file', 'env', 'browser', 'command']
2020-05-20T17:31:59.0354603Z ##[warning]Unexpected input 'wait-on-timeout', valid inputs are ['record', 'config', 'config-file', 'env', 'browser', 'command']
2020-05-20T17:31:59.0359825Z ##[group]Run cypress-io/github-action@v1
2020-05-20T17:31:59.0360019Z with:
2020-05-20T17:31:59.0360195Z   working-directory: ./frontend
2020-05-20T17:31:59.0360371Z   group: chrome-3.6
2020-05-20T17:31:59.0360538Z   browser: chrome
2020-05-20T17:31:59.0360717Z   headless: true
2020-05-20T17:31:59.0360882Z   record: true
2020-05-20T17:31:59.0361041Z   parallel: true
2020-05-20T17:31:59.0361226Z   start: npx ws --spa -s index.html --hostname 0.0.0.0 --port 4200 --directory dist
2020-05-20T17:31:59.0361497Z   wait-on: http://localhost:4200
2020-05-20T17:31:59.0361693Z   wait-on-timeout: 90
2020-05-20T17:31:59.0361861Z   config-file: cypress.json
2020-05-20T17:31:59.0362034Z env:
2020-05-20T17:31:59.0362737Z   AWS_DEFAULT_REGION: ***
2020-05-20T17:31:59.0362934Z   AWS_REGION: ***
2020-05-20T17:31:59.0363121Z   AWS_ACCESS_KEY_ID: ***
2020-05-20T17:31:59.0363342Z   AWS_SECRET_ACCESS_KEY: ***
2020-05-20T17:31:59.0363533Z   pythonLocation: /opt/hostedtoolcache/Python/3.6.10/x64
2020-05-20T17:31:59.0363860Z   CYPRESS_RECORD_KEY: ***
2020-05-20T17:31:59.0364362Z   GITHUB_TOKEN: ***
2020-05-20T17:31:59.0364530Z ##[endgroup]
2020-05-20T17:31:59.1550312Z restoring cache /home/runner/.npm
2020-05-20T17:31:59.1551820Z primary key npm-linux-x64-7c1df3bf3cbc38eae3903f424d78ffc55ade86dd53c19df38b54cd481087f507d9aa58dca1415f40b8ca79c299224b4f9f004e3b98c1b3e99622b246c8fab631
2020-05-20T17:31:59.1750718Z restoring cache /home/runner/.cache/Cypress
2020-05-20T17:31:59.1752246Z primary key cypress-linux-x64-7c1df3bf3cbc38eae3903f424d78ffc55ade86dd53c19df38b54cd481087f507d9aa58dca1415f40b8ca79c299224b4f9f004e3b98c1b3e99622b246c8fab631
2020-05-20T17:32:02.6710220Z [command]/bin/tar -xz -f /home/runner/work/_temp/f0bfa687-51c2-40cb-b219-1bbb4c7a1c31/cache.tgz -C /home/runner/.cache/Cypress
2020-05-20T17:32:03.5418209Z [command]/bin/tar -xz -f /home/runner/work/_temp/92523f23-1a7e-4e20-867e-faec5a8e0a70/cache.tgz -C /home/runner/.npm
2020-05-20T17:32:05.8987991Z Cache restored from key: npm-linux-x64-7c1df3bf3cbc38eae3903f424d78ffc55ade86dd53c19df38b54cd481087f507d9aa58dca1415f40b8ca79c299224b4f9f004e3b98c1b3e99622b246c8fab631
2020-05-20T17:32:15.2990556Z Cache restored from key: cypress-linux-x64-7c1df3bf3cbc38eae3903f424d78ffc55ade86dd53c19df38b54cd481087f507d9aa58dca1415f40b8ca79c299224b4f9f004e3b98c1b3e99622b246c8fab631
2020-05-20T17:32:15.3014138Z [command]/usr/local/bin/npm ci
2020-05-20T17:33:03.6891914Z 
2020-05-20T17:33:03.6892791Z > @angular/cli@9.1.0 postinstall /home/runner/work/Odin/Odin/frontend/node_modules/@angular/cli
2020-05-20T17:33:03.6893146Z > node ./bin/postinstall/script.js
2020-05-20T17:33:03.6893387Z 
2020-05-20T17:33:04.2877487Z 
2020-05-20T17:33:04.2878261Z > fsevents@1.2.9 install /home/runner/work/Odin/Odin/frontend/node_modules/fsevents
2020-05-20T17:33:04.2878610Z > node install
2020-05-20T17:33:04.2878835Z 
2020-05-20T17:33:04.5693643Z 
2020-05-20T17:33:04.5705799Z > core-js@3.6.4 postinstall /home/runner/work/Odin/Odin/frontend/node_modules/@angular-devkit/build-angular/node_modules/core-js
2020-05-20T17:33:04.5706825Z > node -e "try{require('./postinstall')}catch(e){}"
2020-05-20T17:33:04.5707237Z 
2020-05-20T17:33:05.0932871Z 
2020-05-20T17:33:05.0933691Z > cypress@4.5.0 postinstall /home/runner/work/Odin/Odin/frontend/node_modules/cypress
2020-05-20T17:33:05.0934680Z > node index.js --exec install
2020-05-20T17:33:05.0934948Z 
2020-05-20T17:33:05.5462839Z Note: Overriding Cypress cache directory to: /home/runner/.cache/Cypress
2020-05-20T17:33:05.5463382Z 
2020-05-20T17:33:05.5463776Z       Previous installs of Cypress may not be found.
2020-05-20T17:33:05.5465854Z 
2020-05-20T17:33:05.5674461Z 
2020-05-20T17:33:05.5681616Z Cypress 4.5.0 is installed in /home/runner/.cache/Cypress/4.5.0
2020-05-20T17:33:05.5684763Z 
2020-05-20T17:33:06.6256468Z added 2614 packages in 49.519s
2020-05-20T17:33:06.6265648Z start server with command "npx ws --spa -s index.html --hostname 0.0.0.0 --port 4200 --directory dist"
2020-05-20T17:33:06.6266327Z current working directory "./frontend"
2020-05-20T17:33:06.6270811Z [command]/usr/local/bin/npx ws --spa -s index.html --hostname 0.0.0.0 --port 4200 --directory dist
2020-05-20T17:33:06.6271512Z waiting on "http://localhost:4200" with timeout of 90 seconds
2020-05-20T17:33:06.6272634Z Listening on [4mhttp://0.0.0.0:4200[0m
2020-05-20T17:33:07.3331637Z Cypress test command: npx cypress run --headless --record --parallel --group chrome-3.6 --config-file cypress.json --ci-build-id 110630394-3 --browser chrome
2020-05-20T17:33:07.3352829Z [command]/usr/local/bin/npx cypress run --headless --record --parallel --group chrome-3.6 --config-file cypress.json --ci-build-id 110630394-3 --browser chrome
2020-05-20T17:33:18.1292068Z [33mCypress could not verify that this server is running:[39m
2020-05-20T17:33:18.1293958Z [33m[39m
2020-05-20T17:33:18.1294940Z [33m  > [34mhttp://localhost.localdomain:8080[33m[39m
2020-05-20T17:33:18.1295559Z [33m[39m
2020-05-20T17:33:18.1296290Z [33mWe are verifying this server because it has been configured as your `baseUrl`.[39m
2020-05-20T17:33:18.1297417Z [33m[39m
2020-05-20T17:33:18.1298139Z [33mCypress automatically waits until your server is accessible before running tests.[39m
2020-05-20T17:33:18.1298724Z [33m[39m
2020-05-20T17:33:18.1299369Z [33m[90mWe will try connecting to it 3 more times...[33m[39m
2020-05-20T17:33:20.8991833Z [33m[90mWe will try connecting to it 2 more times...[33m[39m
2020-05-20T17:33:24.1314462Z [33m[90mWe will try connecting to it 1 more time...[33m[39m
2020-05-20T17:33:24.1316581Z [33m[90m[33m[39m
2020-05-20T17:33:28.0089545Z [31mCypress failed to verify that your server is running.[39m
2020-05-20T17:33:28.0103059Z [31m[39m
2020-05-20T17:33:28.0105333Z [31mPlease start this server and then run Cypress again.[39m
2020-05-20T17:33:28.2839496Z Error: The process '/usr/local/bin/npx' failed with exit code 1
2020-05-20T17:33:28.2840121Z     at ExecState._setResult (/home/runner/work/_actions/cypress-io/github-action/v1/dist/index.js:1188:25)
2020-05-20T17:33:28.2840632Z     at ExecState.CheckComplete (/home/runner/work/_actions/cypress-io/github-action/v1/dist/index.js:1171:18)
2020-05-20T17:33:28.2841108Z     at ChildProcess.<anonymous> (/home/runner/work/_actions/cypress-io/github-action/v1/dist/index.js:1071:27)
2020-05-20T17:33:28.2841334Z     at ChildProcess.emit (events.js:210:5)
2020-05-20T17:33:28.2841549Z     at maybeClose (internal/child_process.js:1021:16)
2020-05-20T17:33:28.2841757Z     at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)
2020-05-20T17:33:28.2846899Z ##[error]The process '/usr/local/bin/npx' failed with exit code 1
2020-05-20T17:33:28.2966192Z Post job cleanup.
2020-05-20T17:33:28.4168778Z [command]/usr/bin/git version
2020-05-20T17:33:28.4270604Z git version 2.26.2

**Comment by craig-dae** [comment]
All tests are failing now with this error.

**Comment by craig-dae** [comment]
This is a duplicate of https://github.com/cypress-io/github-action/issues/141. Closing in favor of that ticket.

**Comment by DlgSHi** [comment]
Cypress probably didn't resolve your `baseUrl` path. Try to:
- change url to `http://127.0.0.1:4200`
- increase timeout to 120-200s
